scripts/admin/launch-meteor

```diff
@@ -82,15 +82,36 @@ if [ ! -x "$METEOR_WAREHOUSE_DIR/meteor" ]; then
   fi
   echo "Installing a Meteor distribution in your home directory." 1>&2
 
-  # Only show progress bar animations if we have a tty
-  # (Prevents tons of console junk when installing within a pipe)
-  if [[ -t 1 ]]; then
-    curl --progress-bar --fail "$TARBALL_URL" | tar -xzf - -C "$INSTALL_TMPDIR"
-  else
-    curl -s --fail "$TARBALL_URL" | tar -xzf - -C "$INSTALL_TMPDIR"
-  fi
+  TARBALL_FILE="$(dirname "$METEOR_WAREHOUSE_DIR")/.meteor-tarball-tmp"
+  rm -rf "$TARBALL_FILE"
+
+  echo "Downloading Meteor distribution"
+  # keep trying to curl the file until it works (resuming where possible)
+  MAX_ATTEMPTS=99
+  set +e
+  CODE=1
+  ATTEMPTS=0
+  while [ $CODE -ne 0 ] && [ $ATTEMPTS -lt $MAX_ATTEMPTS ]
+  do
+    ATTEMPTS=$((ATTEMPTS + 1))
+
+    # Only show progress bar animations if we have a tty
+    # (Prevents tons of console junk when installing within a pipe)
+    if [[ -t 1 ]]; then
```

Comment:
Yeah, I just sussed out this loader a couple weeks ago, but I incorrectly assumed the shebang was the same as install.meteor.com.  So I guess this is fine – though generally it's best to keep POSIX-compliant – though I find it highly frustrating exactly because of issues like this. :wink: